doc/user/content/self-managed/installation/install-on-aws.md

```diff
@@ -14,151 +23,309 @@ For testing purposes only. For testing purposes only. For testing purposes only.
 
 ## Prerequisites
 
+### Helm 3.2.0+
+
+If you don't have Helm version 3.2.0+ installed, refer to the [Helm
+documentation](https://helm.sh/docs/intro/install/).
+
+### AWS CLI
+
+If you do not have the AWS CLI installed,
+
+- Install the AWS CLI. For details, see the [AWS
+  documentation](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html).
+
+- Configure with your AWS credentials. For details, see the [AWS
```

Comment:
Wasn't sure if we should just give them the steps like `aws configure` or `export AWS_ACCESS_KEY_ID` ... But can, if you all prefer.

Reply:
soft preference for what you have here already! seems much simpler to maintain